test/VariablesTest.cs

```diff
@@ -22,8 +22,8 @@ public void GetVariableTest()
             Assert.AreEqual("testvalue", Client.Variable("foo/bar").GetValue());
 
             // TODO: not sure if this is supposed to be a plus or %20 or either
-            Mocker.Mock(new Uri("test:///secrets/" + TestAccount + "/variable/foo+bar"), "space test");
-            Assert.AreEqual("space test", Client.Variable("foo bar").GetValue());
+            // since we are using EscapeDataString %20 is convert to space, however plus is not converted anymore,
+            Mocker.Mock (new Uri ("test:///secrets/" + TestAccount + "/variable/foo%20bar"), "space test");
```

Comment:
What is this test doing?  It looks like you're just mocking the response to return a variable with a space in it.  It looks like your changes enable variable names with spaces to be correctly handled.

Reply:
This test was updated - Upon creating a variable with space, a URL (of variable path member) was created. Before update, spaces was converted to + and after update, I verify that they are converted to %20